src/server/game/World/World.cpp

```diff
@@ -1123,7 +1123,13 @@ void World::LoadConfigSettings(bool reload)
         sLog->outError("Battleground.ReportAFK (%d) must be <10. Using 3 instead.", m_int_configs[CONFIG_BATTLEGROUND_REPORT_AFK]);
         m_int_configs[CONFIG_BATTLEGROUND_REPORT_AFK] = 3;
     }
-
+    m_int_configs[CONFIG_BATTLEGROUND_RESPAWN_TIME]        = sConfigMgr->GetIntDefault("Battleground.RespawnTime", 30);
+    if (m_int_configs[CONFIG_BATTLEGROUND_RESPAWN_TIME] < 3 || m_int_configs[CONFIG_BATTLEGROUND_RESPAWN_TIME] > 120)
+    {
+        sLog->outError("Battleground.RespawnTime (%i) must be in range 3..120. Using 30 instead.", m_int_configs[CONFIG_BATTLEGROUND_RESPAWN_TIME]),
```

Comment:
```suggestion
        sLog->outError("Battleground.RespawnTime (%i) must be in range 3..120. Using 30 instead.", m_int_configs[CONFIG_BATTLEGROUND_RESPAWN_TIME]);
```

Reply:
pretty sure this should be a comma, semicolon is on the next line

Reply:
always semicolon at the end of a line.

Reply:
nevermind 👍